src/fix_wall_region.cpp

```diff
@@ -70,6 +71,14 @@ FixWallRegion::FixWallRegion(LAMMPS *lmp, int narg, char **arg) :
   sigma = force->numeric(FLERR,arg[6]);
   cutoff = force->numeric(FLERR,arg[7]);
 
+  if (style == MORSE) {
```

Comment:
since you already have to use an if statement here. wouldn't it be better, if the cutoff was the last parameter, similar to how things are set up in the `pair_coeff` command for pair style morse? it would also reduce compiler warnings and simplify memory checking to initialize alpha to 0.0, for wall styles, where it is not used.

Reply:
Well, I did it like this to not change the "signature" of the command too much. If I did it the way you suggest I'd have to have cases by style in the documentation or something like that... it's not really a problem, I just thought this might be more elegant. Do you still want me to change this?
When I think about it, maybe it would be better then to split this fix into multiple fixes, like the other wall fixes (e.g. wall/region/lj126, wall/region/morse, wall/region/colloid etc.)?

Reply:
@erozic i've just implemented the changes i mentioned and pushed the update into your branch. i think consistency is preferable here. and i feel the changes required are not that large anyway.
Please have a look and let us know if you see any problems.
I don't think it would be a good idea to split the file. we have too many files to manage with redundant code already.